test/fixtures/run-script/cannot-parse/package.json

```diff
@@ -0,0 +1 @@
+{ "non-parsable package.json"
```

Comment:
Can you add EOF new line character?

Reply:
Thank you for your review, I committed to reflect your suggestion!